projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.html

```diff
@@ -2,6 +2,27 @@
   <aca-page-layout-header>
     <adf-breadcrumb root="APP.BROWSE.LIBRARIES.MENU.FAVORITE_LIBRARIES.TITLE"> </adf-breadcrumb>
 
+    <div *ngIf="!searchVisibility">
+      <div class="adf-toolbar--spacer adf-toolbar-divider"></div>
+      <span class="action-bar">
+        <button class="aca-mat-button aca-create-button" mat-stroked-button data-automation-id="create-button"
+          [matMenuTriggerFor]="createMenu" [class]="{'app-create-menu-secondary-button': isMainActionPresent}" 
```

Comment:
you copied `isMainActionPresent` from `CreateMenuComponent` but this feature is not used in this component